dangerzone/isolation_provider/base.py

```diff
@@ -93,8 +103,42 @@ def convert(
             self.print_progress(document, True, str(e), 0)
             document.mark_as_failed()
 
-    def doc_to_pixels(
-        self, document: Document, tempdir: str, p: subprocess.Popen
+    def _pixels_to_pdf(
+        self,
+        untrusted_data: bytes,
+        untrusted_width: int,
+        untrusted_height: int,
+        ocr_lang: Optional[str],
+    ) -> fitz.Document:
+        """Convert a byte array of RGB pixels into a PDF page, optionally with OCR."""
+        pixmap = fitz.Pixmap(
+            fitz.Colorspace(fitz.CS_RGB),
+            untrusted_width,
+            untrusted_height,
+            untrusted_data,
+            False,
+        )
+        pixmap.set_dpi(DEFAULT_DPI, DEFAULT_DPI)
+
+        if ocr_lang:  # OCR the document
+            page_pdf_bytes = pixmap.pdfocr_tobytes(
```

Comment:
Well, we don't care about this anymore in our on-host conversion PR, since there's no container stderr to parse, in the second phase.

Reply:
Well, I wouldn't mind that much for the CLI version, because the extra line are not as disttracting as tracebacks. In any case, the new PyMuPDF version should have the necessary fix, so I think we're good.